app/views/curation_concerns/base/_member.html.erb

```diff
@@ -1,19 +0,0 @@
-<tr class="<%= dom_class(member) %> attributes">
-  <td class="thumbnail">
-    <%= link_to image_tag(thumbnail_url(member),
-                    class: "member-item-preview",
-                            alt: "",
-                            role: "presentation"),
-                  "#", 
-                  { "aria-label" => "Load preview image",
-                    "data-target" => "#preview-#{preview_id(member)}",
-                    "data-toggle" => "modal",
-                    title: "Preview image in lightbox" } %>
-  </td>
-  <td class="attribute filename"><%= link_to(member.link_name, contextual_path(member, @presenter)) %></td>
-  <td class="attribute date_uploaded"><%= member.try(:date_uploaded) %></td>
-  <td class="attribute permission"><%= member.permission_badge %></td>
-  <td>
-    <%= render 'actions', member: member %>
-  </td>
-</tr>
```

Comment:
weird, I don't get how this wasn't causing a problem being there!  Maybe we weren't using that partial anymore. Cool.